lib/core/src/Cardano/Wallet/Api/Link.hs

```diff
@@ -633,6 +634,21 @@ getTransaction w t = discriminate @style
     tid = ApiTxId (t ^. typed @(ApiT (Hash "Tx")))
     mkURL mk = mk wid tid
 
+createUnsignedTransaction
```

Comment:
```suggestion
createTransaction
```

Reply:
unfortunately, there is already `createTransaction`